### DISCUSSION TITLE
How to get api_codes ?

### DISCUSSION THREAD

**Discussion by kornflex** [tweepy/tweepy#1741] [Questions] [answered]
Hello,

When I try to retweet already retweeted, I want to catch this error.

So :
```
try:
# retweet a tweet

except tweepy.TweepyException as e:
     if e.api_codes == 327:
     # something 
```

But I never enter in the conditionnal == 327 :/ 
It's an integer so I d'ont know how to catch it :/

Thank you for your help

**Comment by Harmon758** [comment, marked as answer]
[`HTTPException.api_codes`](https://tweepy.readthedocs.io/en/v4.4.0/exceptions.html#tweepy.errors.HTTPException.api_codes) is a list of integers, so you can check for the specific error code within the list, e.g.:
`if 327 in e.api_codes`.
